WalletWasabi.Gui/CommandLine/CommandInterpreter.cs

```diff
@@ -67,23 +69,23 @@ private static void EnsureBackwardCompatibilityWithOldParameters(ref string[] ar
 			args = listArgs.ToArray();
 		}
 
-		private static void ShowVersion()
+		private void ShowVersion()
 		{
 			Console.WriteLine($"Wasabi Client Version: {Constants.ClientVersion}");
 			Console.WriteLine($"Compatible Coordinator Version: {Constants.ClientSupportBackendVersionText}");
 			Console.WriteLine($"Compatible Bitcoin Core and Bitcoin Knots Versions: {Constants.BitcoinCoreVersion}");
 			Console.WriteLine($"Compatible Hardware Wallet Interface Version: {Constants.HwiVersion}");
 		}
 
-		private static void ShowHelp(OptionSet p)
+		private void ShowHelp(OptionSet p)
 		{
 			ShowVersion();
 			Console.WriteLine();
 			Console.WriteLine("Usage: wassabee [OPTIONS]+");
 			Console.WriteLine("Launches Wasabi Wallet.");
 			Console.WriteLine();
 			Console.WriteLine("Options:");
-			p.WriteOptionDescriptions(Console.Out);
+			p.WriteOptionDescriptions(TextWriter);
```

Comment:
This change is perfect however for consistency it has to be applied to the `Console.WriteLine` above too.

Reply:
Fixed